### ISSUE TITLE
announce `accepted completion` to screen reader users

### ISSUE THREAD

**Issue by meganrogge** [microsoft/vscode#195411]
cc @jooyoungseo and @rperez030 

@gdomingu got some feedback that it could be useful to alert when a completion is accepted.

I can imagine why this would be helpful, but also, the user has taken the action and knows what the completion is, so wonder if this would just add noise.

**Comment by jooyoungseo** [comment]
IMO, this is not necessary b/c users  take the accept gesture; thus, the event result is very obvious.

**Comment by meganrogge** [comment]
Agree, it's also clear because the user's focus is in the editor right where the suggestion was inserted.

**Comment by meganrogge** [comment]
Will wait for more feedback, but seems likely we'll close this.

**Comment by rperez030** [comment]
I agree, however that announcement is currently present if I accept a suggestion from the accessible view. I wonder if it should be removed there for consistency. There is a change of context that happens because the accessible view closes and focus goes to the editor, which I believe serves as a confirmation.

**Comment by alexr00** [comment]
It seems like I should verify that a screen reader alert does not happen when accepting a completion in the "accessible view". Which accessible view is this?

**Comment by meganrogge** [comment]
Correct, the inline completions accessible view, so trigger an inline completion, then use alt+f2
